packages/app/src/pages/api/ownership/request/index.ts

```diff
@@ -33,4 +44,44 @@ const handler: NextApiHandler = async (req, res) => {
   }
 };
 
+// TODO: Ajouter le décorateur Staff only
+const get: NextApiHandler = async (req, res) => {
+  const {
+    siren: sirenQuery,
+    status: statusQuery,
+    limit: limitQuery,
+    offset: offsetQuery,
+    orderBy: orderByQuery,
+    orderAsc: orderAscQuery,
+  } = req.query;
+
+  const siren = normalizeQueryParam(sirenQuery);
```

Comment:
Should be used also in controller router if revelant

Reply:
??